design-proposals/virt-operator-deploy-instancetypes.md

```diff
@@ -0,0 +1,97 @@
+# Overview
+
+The [`common-instancetypes`](https://github.com/kubevirt/common-instancetypes)
+project provides a set of useful [instance types and preferences](https://kubevirt.io/user-guide/virtual_machines/instancetypes/) for
+use when creating
+[`VirtualMachines`](http://kubevirt.io/api-reference/main/definitions.html#_v1_virtualmachine)
+within KubeVirt. Until now operators and users could manually deploy these
+resources from the [upstream project
+tree](https://github.com/kubevirt/common-instancetypes#installation), from a
+[specific release](https://github.com/kubevirt/common-instancetypes/releases/tag/v0.3.1)
+or by using the [SSP operator](https://github.com/kubevirt/ssp-operator#ssp-operator).
+
+This proposal suggests moving the deployment of the cluster-wide
+`common-instancetypes` resources to `virt-operator` so that they are
+consistently available across KubeVirt deployments.
+
+## Motivation
+
+The ultimate goal of this proposal is to improve KubeVirt's default user
+experience when creating `VirtualMachines` across several different deployments.
+
+The deployment by default of the cluster-wide `common-instancetypes` resources
+will allow users to rely more freely on the presence of the resources and make
+use of them during their `VirtualMachine` creation flows.
+
+## Goals
+
+* For new deployments, the deployment of the cluster-wide `common-instancetypes`
+  resources by `virt-operator`
```

Comment:
Can an admin opt-out of this?

Reply:
Yes sorry I didn't include this in the design proposal but I want this configurable from the KV CR.